src/frontend/test_runner/tests/testdata/basic_query_2.yaml

```diff
@@ -147,6 +147,26 @@
           StreamProject { exprs: [$2, $0, ($0 + $1), $3], expr_alias: [ ,  ,  ,  ] }
             StreamExchange { dist: HashShard([2]) }
               StreamTableScan { table: t, columns: [v1, v2, v3, _row_id#0], pk_indices: [3] }
+- sql: |
+    create table t(v1 int, v2 int);
```

Comment:
Better to have a comment in SQL on what this test case is testing about.

e.g.,

```
- sql: |
      /* test agg call with complex group by */
     your query
```